.github/pull-request-template.md

```diff
@@ -17,7 +17,7 @@
 Fixes #
 
 <!--
-Please add an entrty to CHANGELOG.adoc file, too, as part of your Pull Request.
+Please add an entry to CHANGELOG.adoc file, too, as part of your Pull Request.
```

Comment:
I know there's some irony that I haven't followed this instruction yet. But I will once the rest of the changes are reviewed.